### PR TITLE
Add Zrok to Proxy section

### PR DESCRIPTION
**Pull request by Nezteb**
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] Format your submission as follows, where `Demo` and `Clients` are optional. Do not add a duplicate `Source code` link if it is the same as the main link. Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) for it, even if the project's webpage or readme uses another capitalisation such as title case, all caps, small caps or all lowercase. `Demo` links should only be used for interactive demos, i.e. not video demonstrations. ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that depend on proprietary services outside the user's control must be marked `⚠`. ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [X] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Miscellaneous`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] If you are adding software forked from another project, please provide/link to a clear list of differences between both.
- [X] The pull request title is informative, unlike "Update README.md". Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb, "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by KieranRobson — APPROVED** [review]
Looks good to me!

**Review by nodiscc — CHANGES_REQUESTED** [review]
The description should explain what the software does.

**Review by nodiscc — APPROVED** [review]
Much better :+1:

File: README.md [hunk 1]
```diff
@@ -1659,6 +1659,7 @@ _Related: [Web Servers](#web-servers)_
 - [Squid](http://www.squid-cache.org/) - Caching proxy for the Web supporting HTTP, HTTPS, FTP, and more. It reduces bandwidth and improves response times by caching and reusing frequently-requested web pages. ([Source Code](https://code.launchpad.net/squid)) `GPL-2.0` `C`
 - [SWAG (Secure Web Application Gateway)](https://github.com/linuxserver/docker-swag) - Nginx webserver and reverse proxy with PHP support, built-in Certbot (Let's Encrypt) client and fail2ban integration. `GPL-3.0` `Docker`
 - [Tinyproxy](https://tinyproxy.github.io/) - Light-weight HTTP/HTTPS proxy daemon. ([Source Code](https://github.com/tinyproxy/tinyproxy)) `GPL-2.0` `C`
+- [zrok](https://zrok.io/) - Open source serveo/ngrok alternative. Creates network tunnels that allow sharing locally running services and other private resources with others over the public Internet using OpenZiti. ([Source Code](https://github.com/openziti/zrok)) `Apache-2.0` `Go`
```
Comment (nodiscc) [inline README.md:1662]: Minor change to be more consistent with other entries:

```suggestion
- [zrok](https://zrok.io/) - Open source . Creates network tunnels that allow sharing locally running services and other private resources with others over the public Internet using OpenZiti (alternative to serveo/ngrok). ([Source Code](https://github.com/openziti/zrok)) `Apache-2.0` `Go`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by nodiscc** [comment]
Hi,
thank you for your contribution. Today an improved version of awesome-selfhosted goes live at https://awesome-selfhosted.net/. Please check the [release announcement](https://github.com/awesome-selfhosted/awesome-selfhosted/releases/tag/1.0.0) for more details.

Contributions should now be made through https://github.com/awesome-selfhosted/awesome-selfhosted-data. If you are still interested in having this software added to the list, please check contribution guidelines there, and open a new PR/issue.

Sorry for the inconvenience.

**Comment by Nezteb** [comment]
> Hi, thank you for your contribution. Today an improved version of awesome-selfhosted goes live at https://awesome-selfhosted.net/. Please check the [release announcement](https://github.com/awesome-selfhosted/awesome-selfhosted/releases/tag/1.0.0) for more details.

Neat!

I hope you don't mind that [I posted about this on the /r/selfhosted subreddit](https://www.reddit.com/r/selfhosted/comments/15y7y36/the_awesomeselfhosted_repo_released_a_new_website/). 😄

**Pull request closed without merging**